src/main/java/org/sonar/plugins/stash/issue/StashComment.java

```diff
@@ -53,6 +63,27 @@ public long getVersion() {
     return version;
   }
   
+  public List<StashTask> getTasks() {
+    return tasks;
+  }
+  
+  public void addTask(StashTask task){
+    tasks.add(task);
+  }
+  
+  public boolean containsNotDeletableTasks() {

```

Comment:
Ambiguity probably comes from wrong naming of the method "containsNotDeletableTasks".
This method will be renamed as "containsPermanentTasks" to make more readable this code.